test/Common.FunctionalTests/Utilities/IISApplication.cs

```diff
@@ -205,7 +205,12 @@ private void RestoreAppHostConfig()
                     retryDelayMilliseconds: 100);
             }
 
-            File.Move(_apphostConfigBackupPath, _apphostConfigPath);
+            RetryHelper.RetryOperation(
+                () => File.Move(_apphostConfigBackupPath, _apphostConfigPath),
+                e => _logger.LogError($"Failed to backup apphost.config: {e.Message}"),
+                retryCount: 3,
```

Comment:
Add a helper, 3 and 100 are copied all over.

Reply:
I wanted to, but it felt weird making a helper of a helper 😅